nautobot/docs/configuration/required-settings.md

```diff
@@ -157,15 +157,17 @@ CACHEOPS_REDIS = False
 
 # If you want to use Sentinel, specify this variable
```

Comment:
I wonder if it'd be better to remove this entire example block of code from here, given that the other file is so much more comprehensive now?